### ISSUE TITLE
Replacing the obsolete applymap with map

### ISSUE THREAD

**Issue by quant12345** [unit8co/darts#2435]
[DataFrame.applymap](https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.applymap.html) has been deprecated.

I'd like to do a PR to fix this.
What does the community think?

Warning:
```
darts/tests/test_timeseries_multivariate.py::TestTimeSeries::test_map

  /darts/darts/tests/test_timeseries.py:1449: FutureWarning: DataFrame.applymap has been deprecated. Use DataFrame.map instead.
    df_0[["0"]] = df_0[["0"]].applymap(fn)

darts/tests/test_timeseries.py::TestTimeSeries::test_map
darts/tests/test_timeseries_multivariate.py::TestTimeSeries::test_map
  /darts/darts/tests/test_timeseries.py:1450: FutureWarning: DataFrame.applymap has been deprecated. Use DataFrame.map instead.
    df_2[["2"]] = df_2[["2"]].applymap(fn)

darts/tests/test_timeseries.py::TestTimeSeries::test_map
darts/tests/test_timeseries_multivariate.py::TestTimeSeries::test_map
  /darts/darts/tests/test_timeseries.py:1451: FutureWarning: DataFrame.applymap has been deprecated. Use DataFrame.map instead.
    df_01[["0", "1"]] = df_01[["0", "1"]].applymap(fn)

darts/tests/test_timeseries.py::TestTimeSeries::test_map
darts/tests/test_timeseries_multivariate.py::TestTimeSeries::test_map
  /darts/darts/tests/test_timeseries.py:1452: FutureWarning: DataFrame.applymap has been deprecated. Use DataFrame.map instead.
    df_012 = df_012.applymap(fn)
```

**Comment by madtoinou** [comment]
Hi @quant12345,

Sure, you can go ahead :)

**Comment by quant12345** [comment]
Hi @madtoinou,
contribution says that you need an issue first, so I asked)

**Comment by dennisbader** [comment]
Fixed in #2436
